File: Q8.py
idade = list(range(20))
vezes = 1
soma = 0 
for i in range(0, len(idade)):
    idade[i] = int(input(f"Digite a {vezes}ª idade: "))
    soma = soma + idade[i]
    vezes += 1
print(f"A soma das 20 idades que você digitou é {soma} anos.")

File: Q8.cpp
#include <iostream>
#include <stdio.h>

using namespace std;

int main()
{
    int idade[20], soma = 0, vezes = 1;
    for (int i = 0; i < sizeof idade / sizeof idade[0]; i ++, vezes ++)
    {
        printf("Digite a %dª idade: ", vezes);
        cin >> idade[i];
        soma = soma + idade[i];
    }
    printf("A soma das 20 idades que você digitou é %d anos.\n", soma);
    return 0;
}